theme/js/components/search/suggestor.vue

```diff
@@ -83,6 +84,7 @@ export default {
   },
   props: {
     onChange: Function,
+    suggestApi: Function,
```

Comment:
Does an Object with a default value work instead of the check l.141. ?

I don't know if `this` is available here, otherwise we can export api from `js/plugins/api`. 
```suggestion
    suggestApi: {
        type: Object,
        default: () => this.$api,
    },
```

Reply:
Thank you for this input, indeed it would be nicer to have a default value here.
However, we're switching back to apiv1 for suggest endpoints (to prevent breaking changes), so we won't need the `suggestApi` prop anymore.